web/src/redux/selectors/user.selector.js

```diff
@@ -31,11 +31,13 @@ export const getUserStateOrTerritory = ({
 export const getUserAffiliationForCurrentState = ({
   user: { data: { state: { id = null } = {}, states = [] } = {} } = {}
 }) => {
-  return {status: states[id]};
+  return { status: states[id] };
 };
 
 export const getUserStateOrTerritoryStatus = state => {
+  console.log({ state });
```

Comment:
flagging these in case you don't want to keep them. No issue on my end if we want to keep them in for easier debugging